VS_Code/README.md

```diff
@@ -0,0 +1,73 @@
+# Visual Studio Code
```

Comment:
Sorry: I just merge @bushidocodes work on vscode. You can add/modify that.

Sorry I'm causing you issues here!

Reply:
No worries. That was a brain fade on my part. I will make the necessary changes.

Reply:
In commit bd616f5